cli/cmd/testdata/install_controlplane_tracing_output.golden

```diff
@@ -852,7 +852,7 @@ data:
       prometheusUrl: ""
       proxy:
         capabilities: null
-        component: linkerd-controller
+        component: ""
```

Comment:
To avoid this it might be fine to remove `Component` from the `Proxy` struct in `values.go`, just as there isn't a `WorkloadKind` entry there.

Reply:
Yep, added this. This required an extra check in `_proxy.tpl` about the existance of `.component`